common/templates/recordAction-ent.html

```diff
@@ -0,0 +1,20 @@
+<div ng-class="{row:rowStyle}">
```

Comment:
`rowstyle` is a `bool` which sets the `row` class of `bootstrap`. When this is set to `true` it keeps action bar on top line and table just below it. When `false` action bar appear in the same line as markdown bubbles.